jsonthebuilder/src/App.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './App.css';
+import ResumeContainer from './components/ResumeContainer';
+
+function App() {
+  return (
+    <>
+      <ResumeContainer />
+    </>
```

Comment:
Since you are just returning one component, you don't actually need the Fragment here!